Commands/Basic/help.py

```diff
@@ -0,0 +1,33 @@
+# /Commands/Basic/help.py
```

Comment:
You should use all lowercase folder names

Reply:
fixed